randovania/games/prime1/json_data/Phendrana Drifts.txt

```diff
@@ -1052,7 +1052,9 @@ Extra - aabb: [121.6723, -738.96625, 43.18039, 297.53424, -556.542, 117.33082]
   > Event - Thardus
       All of the following:
           Shoot Any Beam
-          Thermal Visor or Invisible Objects (Beginner)
+          Any of the following:
+              Thermal Visor or Invisible Objects (Beginner)
+              No Combat Visor and Thermal Visor and No X-Ray Visor and Invisible Objects (Intermediate)
```

Comment:
Did you test this out in-game? Aren't you like completely blind? Or just a little blind?